src/variants.js

```diff
@@ -55,89 +97,122 @@ export async function getVariant(name) {
 	return variants.find(variant => variant.name === name);
 }
 
-export const shortForms = /** @type {const} */ ({
-	'Red': 'r',
-	'Yellow': 'y',
-	'Green': 'g',
-	'Blue': 'b',
-	'Purple': 'p',
-	'Teal': 't',
-	'Black': 'k',
-	'Rainbow': 'm',
-	'White': 'w',
-	'Pink': 'i',
-	'Brown': 'n',
-	'Omni': 'o',
-	'Null': 'u',
-	'Prism': 'i'
-});
+export let shortForms = /** @type {string[]} */ (['r', 'y', 'g', 'b', 'p']);
+
+/**
+ * Edits shortForms to have the correct acryonyms.
+ * @param {Variant} variant
+ */
+export async function getShortForms(variant) {
+	const colors = await colours_promise;
```

Comment:
There's still a couple `color`'s scattered around, you should be able to use Find in Files to get them all.

Reply:
oops